client/src/components/History/Content/ContentItem.vue

```diff
@@ -18,22 +18,21 @@
                         <font-awesome-icon class="text-info" icon="arrow-circle-up" />
                     </span>
                     <span
-                        v-else-if="highlight == 'noInputs'"
+                        v-else-if="highlight == 'active'"
                         v-b-tooltip.hover
-                        title="No Inputs for this item"
+                        title="Inputs/Outputs highlighted for this item"
                         tabindex="0"
                         @click.stop="toggleHighlights"
                         @keypress="toggleHighlights">
-                        <font-awesome-icon icon="minus-circle" />
+                        <font-awesome-icon icon="check-circle" />
                     </span>
                     <span
                         v-else-if="highlight == 'output'"
                         v-b-tooltip.hover
-                        title="Inputs highlighted for this item"
+                        title="Output"
                         tabindex="0"
-                        @click.stop="toggleHighlights"
-                        @keypress="toggleHighlights">
-                        <font-awesome-icon icon="check-circle" />
+                        @click.stop="toggleHighlights">
+                        <font-awesome-icon class="text-info" icon="arrow-circle-down" />
                     </span>
```

Comment:
Not fully keyboard accessible. To avoid this, use buttons for clickable elements instead:

```vue
<b-button
    v-if="highlight === 'input'"
    v-b-tooltip.hover
    variant="link"
    class="p-0"
    title="Input"
    @click.stop="toggleHighlights">
    <font-awesome-icon class="text-info" icon="arrow-circle-up" />
</b-button>
<b-button
    v-else-if="highlight === 'active'"
    v-b-tooltip.hover
    variant="link"
    class="p-0"
    title="Inputs/Outputs highlighted for this item"
    @click.stop="toggleHighlights">
    <font-awesome-icon icon="check-circle" />
</b-button>
<b-button
    v-else-if="highlight === 'output'"
    v-b-tooltip.hover
    variant="link"
    class="p-0"
    title="Output"
    @click.stop="toggleHighlights">
    <font-awesome-icon class="text-info" icon="arrow-circle-down" />
</b-button>
```

Reply:
Thank you! i will correct this